package.json

```diff
@@ -125,6 +125,10 @@
       "packages/fluentui/*"
     ],
     "nohoist": [
+      "@fluentui/make-styles/stylis",
```

Comment:
I just found out that our current nohoist definitions dosn't work.

- this explicitly turns of nohoist for all packages using stylis. in next PR we can apply single version policy by adding latest stylis to root.
- this change also reverts needs of any version changes that affects consumers of make styles